notebooks/002-openvino-api/002-openvino-api.ipynb

```diff
@@ -638,28 +653,25 @@
     "import time\n",
     "from pathlib import Path\n",
     "\n",
-    "from openvino.runtime import Core, PartialShape\n",
+    "from openvino.runtime import Core\n",
     "\n",
     "ie = Core()\n",
     "\n",
-    "device_name = \"GPU\"  # Model Caching is not available for CPU\n",
+    "device_name = \"GPU\" \n",
     "\n",
-    "if device_name in ie.available_devices and device_name != \"CPU\":\n",
```

Comment:
This if should be here as there could be a situation the user doesn't have GPU.

Reply:
Sure. I have included the if to cover the scenario.